examples/plotting_basemap_background.py

```diff
@@ -31,38 +31,31 @@
 df = df.to_crs(epsg=3857)
 
 ###############################################################################
-# Contextily helper function
-# ==========================
-#
-# We define a small helper function that uses
-# `contextily <https://github.com/darribas/contextily>`__ to add a map
-# as background to an existing plot: 
 
 import contextily as ctx
 
-def add_basemap(ax, zoom, url='http://tile.stamen.com/terrain/tileZ/tileX/tileY.png'):
-    xmin, xmax, ymin, ymax = ax.axis()
-    basemap, extent = ctx.bounds2img(xmin, ymin, xmax, ymax, zoom=zoom, url=url)
-    ax.imshow(basemap, extent=extent, interpolation='bilinear')
-    # restore original x/y limits
-    ax.axis((xmin, xmax, ymin, ymax))
-
 ###############################################################################
 # Add background tiles to plot
 # ============================
 #
-# Now we can use the above function to easily add a background map to our
-# plot. The `zoom` keyword is required and let's you specify the detail of the
-# map tiles (be careful to not specify a too high `zoom` level, as this can
-# result in a large download):
+# We can use `add_basemap` function of contextily to easily add a background
+# map to our plot. :
+
+ax = df.plot(figsize=(10, 10), alpha=0.5, edgecolor='k')
+ctx.add_basemap(ax)
+
+###############################################################################
+# We can control the detail of the map tiles using the optional `zoom` keyword
+# (be careful to not specify a too high `zoom` level,
+# as this can result in a large download).:
 
 ax = df.plot(figsize=(10, 10), alpha=0.5, edgecolor='k')
-add_basemap(ax, zoom=10)
+ctx.add_basemap(ax, zoom=12)
```

Comment:
The original example of zoom=10 gave a less detailed background, with zoom=12 it actually takes some time ..

Do you think the higher zoom is better to illustrate? (if so, the longer doc build time is fine for me)

Reply:
Yes. In reality, you'll rather want more detailed map than blurry one. I tried to simulate real-life use. I am aware that it takes longer, but I feel it makes more sense this way.